google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/BigtableBatchingCallSettings.java

```diff
@@ -55,6 +61,9 @@
   // This settings is just a simple wrapper for BatchingCallSettings to allow us to add
   // additional functionality.
   private BatchingCallSettings<RowMutationEntry, Void, BulkMutation, Void> batchingCallSettings;
+  private boolean isLatencyBasedThrottlingEnabled;
+  private Long targetRpcLatencyMs;
+  private DynamicFlowControlSettings dynamicFlowControlSettings;
```

Comment:
please make these final.
Also if you dont mind, please backfill final for batchingCallSettings